biblio.module

```diff
@@ -408,3 +408,14 @@ function biblio_get_author_roles($code = FALSE) {
     return $mapping;
   }
 }
+
+/**
+ * Implements hook_config_info().
+ */
+function biblio_config_info() {
+  $prefixes['biblio.settings'] = array(
+    'label' => t('Biblio general settings'),
+    'group' => t('Biblio'),
```

Comment:
General settings go in the `Configuration` group. These are usually only classified as their own group if there are a set of prefixes such as 'Image styles' or 'Views'.

Reply:
@jenlampton I followed the docs in https://docs.backdropcms.org/converting-modules-from-drupal#declare-module-config, so maybe we should update that as well to invite people to use `Configuration` instead of the name of the module.

Reply:
I updated https://docs.backdropcms.org/converting-modules-from-drupal :)